ChopChop/Models/Recipe/Recipe/Recipe.swift

```diff
@@ -0,0 +1,210 @@
+import Foundation
+import GRDB
+
+/// Note there is no relationship between steps and ingredients after parsing stage
+class Recipe: FetchableRecord {
+    var id: Int64?
+    private(set) var name: String
+    private(set) var servings: Double
```

Comment:
Does this refer to how many people the recipe can serve? In this case should it be a Int, unless u want to handle cases like 2-3 people

Reply:
I think recipes can have 2.5 servings etc. Or maybe the person is not very hungry and he just wants to cook 0.5 servings